### DISCUSSION TITLE
Zoom Phone API support

### DISCUSSION THREAD

**Discussion by pzelenko** [Jericho/ZoomNet#232] [Q&A]
Hi Jericho, All,

We would like to propose contribution to extending the library by adding Zoom Phone API support.
What do you think about this, and whether this is inline with your future roadmap?

Another approach could be adding custom request execution support in library, which can be used to send Zoom Phone API requests using this custom execution method from library.

Thanks in advance

**Comment by Jericho** [comment]
I welcome contributions and would love to be able to add support for Zoom's Phone api to the library. In fact, [I created an issue](https://github.com/Jericho/ZoomNet/issues/22) a while back to track this enhancement.

Quick observation: I estimate that there are more than 100 methods associated with the Phone resource such as "Get an audio item", "List auto receptionists", "List blocked lists", "Get call handling settings", "Get call log details", etc. It's quite an undertaking!
